CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
+cmake_minimum_required(VERSION 3.13)
+cmake_policy(SET CMP0167 NEW)
```

Comment:
This will fail on older CMake versions. You can't set policies for unknown policies.